service/build.gradle.kts

```diff
@@ -0,0 +1,92 @@
+plugins {
+    kotlin("jvm")
+    kotlin("plugin.spring")
+    id("org.springframework.boot") version "2.3.7.RELEASE"
+}
+
+
+//////////////////////////
+// componentTest source set
+//////////////////////////
+
+sourceSets.create("componentTest") {
+    withConvention(org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet::class) {
+        kotlin.srcDir("src/componentTest/kotlin")
+        resources.srcDir("src/componentTest/resources")
+        compileClasspath += sourceSets["main"].output + configurations["testRuntimeClasspath"]
+        runtimeClasspath += output + compileClasspath + sourceSets["test"].runtimeClasspath
+    }
+}
+
+val componentTest = task<Test>("componentTest") {
+    description = "Runs the component tests"
+    group = "verification"
+    testClassesDirs = sourceSets["componentTest"].output.classesDirs
+    classpath = sourceSets["componentTest"].runtimeClasspath
+    testLogging.showStandardStreams = true
+    mustRunAfter(tasks["test"])
+}
+
+val componentTestImplementation: Configuration by configurations.getting {
+    extendsFrom(configurations.testImplementation.get())
+}
+val componentTestRuntimeOnly: Configuration by configurations.getting {
+    extendsFrom(configurations.testRuntimeOnly.get())
+}
+
+configurations["componentTestImplementation"].extendsFrom(configurations.runtimeOnly.get())
+
+tasks.check { dependsOn(componentTest) }
+
+tasks.withType<Test> {
+    systemProperty("yatspec.output.dir", "$buildDir/reports/yatspec")
+    useJUnitPlatform()
+}
+
+
+//////////////////////////
+// dependencies
+//////////////////////////
+
+dependencies {
+    implementation(project(":api"))
+    implementation("org.springframework.boot:spring-boot-starter-actuator")
+    implementation("org.springframework.cloud:spring-cloud-starter-openfeign")
+    implementation("org.springframework.boot:spring-boot-starter-data-rest")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+
+    //////////////////////////////////
+    // Component test dependencies
+    componentTestImplementation("org.springframework.boot:spring-boot-starter-test")
+
+    componentTestImplementation("com.github.nickmcdowall:yatspec:223")
```

Comment:
guessing this is a locally generated version? Can switch to latest publicly available version: `2021.0.1`